VideoWeb/VideoWeb/ClientApp/src/app/services/device-type.service.spec.ts

```diff
@@ -97,6 +97,22 @@ describe('DeviceType', () => {
         expect(service.getOSVersion()).toBe(testOsVersion);
     });
 
+    it('should return whether hte OS is iOS', () => {
```

Comment:
Minor. Misspelling of "the" don't worry about it if there's no other changes to make